CONTRIBUTING.md

```diff
@@ -127,14 +124,19 @@ git commit --amend
 git push origin my-feature-branch -f
 ```
 
-#### Check on Your Pull Request
+## Check on Your Pull Request
 
-Go back to your pull request after a few minutes and see whether it passed muster with Travis-CI. Everything should look green, otherwise fix issues and amend your commit as described above.
+Go back to your pull request after a few minutes and see whether it passed
+muster with Travis-CI. Everything should look green, otherwise fix issues and
```

Comment:
`s/Travis-CI/GitHub Actions/`.  However, considering this targets people who have apparently never used git, expecting them to already be aware of CI doesn't make much sense.

Reply:
```suggestion
tests with GitHub Actions. Everything should look green, otherwise fix issues and
```